### ISSUE TITLE
Greetings

### ISSUE THREAD

**Issue by Pabzdabeast** [sundowndev/ama#3]
I am a student in the Netherlands, and I am writing about your software. I wanted to know if I could ask you certain questions. When did you develop this software? and was there a purpose for the development? Thank you in advance!

**Comment by Pabzdabeast** [comment]
I was writing about PhoneInfoga*

**Comment by sundowndev** [comment]
Hi, I'm glad you're asking :) 

> When did you develop this software? 

I started in late 2018 when I was in vacation in Reunion Island, I was bored so decided to start a new Python project. I also started to write an article about it on the side, so the design of the program evolved more and more over time until I reached v1 which was the first mature version.

> and was there a purpose for the development? 

The purpose was to learn about OSINT and Python, as well as having open source stuff to show off. A few weeks before I began that project, I was looking for an internship, and a company hired me thanks to my GitHub profile and my activity. So this is what I've learned so far with this project :

- Always start with v0, considering your software design as immature and instable, v0 is the prototyping phase
- OSS helps a lot to find tech jobs
- Writing articles on your findings or projects amplifies the attention given to it and helps others to understand how you resolved problems
- Good documentation is key to the success of your OSS project

In January 2020, I decided to start learning Go, so my one of my first Go projects was to migrate PhoneInfoga to Go. And I'm not disappointed at all!

Feel free to ask other questions, and send the link to your article later!
